ApollosProject/apollos-prototype
# Improves consistency of tab view fix

## DESCRIPTION

### What does this PR do, or why is it needed?

Previously, we were importing a prop override into the exported `TabView` component inside `apolloschurchapp`. This mean that churches that used a different version of the `TabView` package might not have access to the same `TabView` that core was using, and have trouble implementing the override. 

By moving the import and usage of the alternative `Pager`, we can ensure that by default the `TabView` and the `Pager` come from the same version of the `react-native-tab-view` package.

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Open a devotional on Android
2. Change tabs by swiping back and forth
3. Make sure you can still change tabs by tapping on the taab headers.

### What automated tests did you write?

Snaps changed because of how third party modules are resolved in `jest`. 

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
